docs/_deployments/serverless-framework.md

```diff
@@ -0,0 +1,342 @@
+---
+title: Serverless Framework
+order: 0
+slug: serverless-framework
+lang: en
+layout: tutorial
+permalink: /deployments/serverless-framework
+redirect_from:
+  - /deployments
+---
+# Deploying with the Serverless Framework
+
+<div class="section-content">
+This guide walks you through preparing and deploying a Slack app using [Bolt for JavaScript][bolt-js], the [Serverless Framework][serverless-framework], and [AWS Lambda][aws-lambda].
+</div>
+
+When you’re finished, you’ll have this ⚡️[Getting Started with Serverless framework app][getting-started-with-serverless-framework-app] to run, modify, and make your own.
```

Comment:
This link will work after we merged this pull request. Until then you can view the app at:
https://github.com/mwbrooks/bolt-js/tree/220-faas-deployment-guides/examples/getting-started-with-serverless-framework